src/test/TxTests.cpp

```diff
@@ -497,7 +497,7 @@ closeLedgerOn(Application& app, uint32 ledgerSeq, TimePoint closeTime,
     TxSetFrameConstPtr txSet;
     if (strictOrder)
     {
-        txSet = TxSetFrame::makeFromTransactions(txs, app, 0, 0, &txs);
+        txSet = TxSetFrame::makeFromTransactions(txs, app, 0, 0, true);
```

Comment:
this should be `strictOrder`, right?

Reply:
We're already in `strictOrder` branch here, so it shouldn't really matter.